src/components/views/avatars/BaseAvatar.tsx

```diff
@@ -66,11 +71,26 @@ const calculateUrls = (url: string, urls: string[], lowBandwidth: boolean): stri
     return Array.from(new Set(_urls));
 };
 
-const useImageUrl = ({ url, urls }): [string, () => void] => {
+/**
+ * Hook for cycling through a changing set of images.
+ *
+ * The set of images is updated whenever `url` or `urls` change, the user's
+ * `lowBandwidth` preference changes, or the client reconnects.
+ *
+ * Returns `[imageUrl, onError]`. When `onError` is called, the next image in
+ * the set will be displayed.
+ */
+const useImageUrl = ({
+    url,
+    urls,
+}: {
+    url: string | undefined;
+    urls: string[] | undefined;
+}): [string | undefined, () => void] => {
     // Since this is a hot code path and the settings store can be slow, we
     // use the cached lowBandwidth value from the room context if it exists
     const roomContext = useContext(RoomContext);
-    const lowBandwidth = roomContext ? roomContext.lowBandwidth : SettingsStore.getValue("lowBandwidth");
+    const lowBandwidth = roomContext.lowBandwidth;
```

Comment:
Why the change here?

Reply:
`useContext` always returns a value, even if it's just the default. So here `roomContext` is `IRoomState`, not `IRoomState | undefined`.